cellpack/autopack/Gradient.py

```diff
@@ -332,9 +337,12 @@ def build_axis_weight_map(self, bb, master_grid_positions, axis="X"):
 
     def set_weights_by_mode(self):
         scaled_distances = self.distances
-        if max(scaled_distances) > 1.0:
-            self.log.error("MAX TOO BIG", max(scaled_distances))
-            # raise ValueError("distances have not been scaled to be from 0.0 to 1.0")
+
+        # TODO: change this back to check distances are assigned correctly
+        scaled_distances[scaled_distances > 1] = 1
+        scaled_distances[scaled_distances < 0] = 0
+        # if max(scaled_distances) > 1.0:
+        #     self.log.error("MAX TOO BIG", max(scaled_distances))
```

Comment:
why is this needed here? Shouldn't they already be scaled?

Reply:
Good catch. Removed this and added an error check.